timeseries/src/autogluon/timeseries/models/local/statsforecast.py

```diff
@@ -491,8 +489,7 @@ def _predict_with_local_model(
         return pd.DataFrame(predictions)
 
 
-# TODO: Starting from StatsForecast v1.5.0, AutoCES can inherit from AbstractProbabilisticStatsForecastModel
-class AutoCESModel(AbstractConformalizedStatsForecastModel):
+class AutoCESModel(AbstractProbabilisticStatsForecastModel):
```

Comment:
Should we also merge `AbstractProbabilisticStatsForecastModel` and `AbstractStatsForecastIntermittentDemandModel` into a single class since no other models inherit from `AbstractConformalizedStatsForecastModel`?

Reply:
You mean merge Conformalized and Intermittent? I would rather keep as is. We can still introduce other Conformalized models that are not intermittent demand models (and do not do zero-clipping).